Extensions/TiledSpriteObject/Extension.cpp

```diff
@@ -210,6 +210,13 @@ void DeclareTiledSpriteObjectExtension(gd::PlatformExtension& extension) {
       .MarkAsAdvanced()
       .SetFunctionName("GetXOffset");
 
+  obj.AddExpression("XOffset",
+                    _("Get X offset"),
+                    _("Get the X offset of the texture of the tiled sprite."),
+                    _("Image offset"),
+                    "res/actions/scaleWidth_black.png")
+      .AddParameter("object", _("Object"), "TiledSprite");
```

Comment:
To be consistent with the condition.
```suggestion
  obj.AddExpression("XOffset",
                    _("Image X Offset"),
                    _("Return the offset used on the X axis when displaying the image."),
                    _("Image offset"),
                    "res/actions/scaleWidth_black.png")
      .AddParameter("object", _("Object"), "TiledSprite");
```